src/components/VmDetails/cards/DetailsCard/index.js

```diff
@@ -691,7 +690,8 @@ class DetailsCard extends React.Component {
     const osType = vm.getIn(['os', 'type'])
     const osId = operatingSystems && operatingSystems.find(os => os.get('name') === osType).get('id')
 
-    const { unit: memoryUnit, value: memorySize } = convertValue('B', vm.getIn(['memory', 'total']))
```

Comment:
I think it will be better leave this line, and change units to `MiB` only in edit mode.

Reply:
this line causes all memory to be rounded to GB during edit. for example: if only edit mode is set to MiB,  1023, 1024 and 1050 MB are all rounded to 1GB, which is inaccurate

Reply:
Yeah, but if we want to be 100% accurate we should use bytes =) Actually we already using formatting with precision in utilization cards, so you'll get not `1 GiB` for `1023 MiB`, but `0.99 GiB`.
https://github.com/oVirt/ovirt-web-ui/blob/e83d2f9357104f988a42168d7cd6ed6e0115cdea/src/components/VmDetails/cards/UtilizationCard/MemoryCharts.js#L39-L41

Reply:
Noted